src/Microsoft.Data.SqlClient/ref/Microsoft.Data.SqlClient.cs

```diff
@@ -0,0 +1,2569 @@
+        [System.ComponentModel.DisplayNameAttribute("Command Timeout")]
+        [System.ComponentModel.RefreshPropertiesAttribute(System.ComponentModel.RefreshProperties.All)]
+        public int CommandTimeout { get { throw null; } set { } }
+#if NETFRAMEWORK
+        /// <include file='../../../doc/snippets/Microsoft.Data.SqlClient/SqlConnectionStringBuilder.xml' path='docs/members[@name="SqlConnectionStringBuilder"]/ConnectionReset/*'/>
+        [System.ComponentModel.BrowsableAttribute(false)]
+        [System.ComponentModel.DisplayNameAttribute("Connection Reset")]
+        [System.ComponentModel.RefreshPropertiesAttribute(System.ComponentModel.RefreshProperties.All)]
+        [System.ObsoleteAttribute("ConnectionReset has been deprecated.  SqlConnection will ignore the 'connection reset' keyword and always reset the connection.")]
+        public bool ConnectionReset { get { throw null; } set { } }
+#endif // NETFRAMEWORK
+        /// <include file='../../../doc/snippets/Microsoft.Data.SqlClient/SqlConnectionStringBuilder.xml' path='docs/members[@name="SqlConnectionStringBuilder"]/ConnectRetryCount/*'/>
+        [System.ComponentModel.DisplayNameAttribute("Connect Retry Count")]
+        [System.ComponentModel.RefreshPropertiesAttribute(System.ComponentModel.RefreshProperties.All)]
+        public int ConnectRetryCount { get { throw null; } set { } }
+        /// <include file='../../../doc/snippets/Microsoft.Data.SqlClient/SqlConnectionStringBuilder.xml' path='docs/members[@name="SqlConnectionStringBuilder"]/ConnectRetryInterval/*'/>
+        [System.ComponentModel.DisplayNameAttribute("Connect Retry Interval")]
+        [System.ComponentModel.RefreshPropertiesAttribute(System.ComponentModel.RefreshProperties.All)]
+        public int ConnectRetryInterval { get { throw null; } set { } }
+        /// <include file='../../../doc/snippets/Microsoft.Data.SqlClient/SqlConnectionStringBuilder.xml' path='docs/members[@name="SqlConnectionStringBuilder"]/ConnectTimeout/*'/>
+        [System.ComponentModel.DisplayNameAttribute("Connect Timeout")]
+        [System.ComponentModel.RefreshPropertiesAttribute(System.ComponentModel.RefreshProperties.All)]
+        public int ConnectTimeout { get { throw null; } set { } }
+#if NETFRAMEWORK
+        /// <include file='../../../doc/snippets/Microsoft.Data.SqlClient/SqlConnectionStringBuilder.xml' path='docs/members[@name="SqlConnectionStringBuilder"]/ContextConnection/*'/>
+        [System.ComponentModel.DisplayNameAttribute("Context Connection")]
+        [System.ComponentModel.RefreshPropertiesAttribute(System.ComponentModel.RefreshProperties.All)]
+        [System.ObsoleteAttribute("ContextConnection has been deprecated.  SqlConnection will ignore the 'Context Connection' keyword.")]
+        public bool ContextConnection { get { throw null; } set { } }
+#endif // NETFRAMEWORK
```

Comment:
All `#if NETFRAMEWORK` blocks in this type can be placed together and combined into 1 block:
- ConnectionReset
- ContextConnection
- NetworkLibrary
- TransparentNetworkIPResolution